routes/listproducts.js

```diff
@@ -8,8 +8,17 @@ listProducts.get('/products', async (req, res) => {
   const count = req.query.count || 5;
   const page = req.query.page || 1;
   const params = [count, (page - 1) * count];
-  const products = await client.query(listProductsQuery, params);
-  res.send(products.rows);
+  await client.query(listProductsQuery, params)
+    .then(data => {
+      res.status(200);
+      res.send(data.rows);
+    }
+    )
+    .catch(err => {
+      res.status(404);
```

Comment:
Typically, server errors result in 500, not 404. Maybe you could put in a special check to make sure it's the right kind of error, in which case you return 404, otherwise default to 500?